cpp/test/IceSSL/configuration/Makefile.mk

```diff
@@ -7,14 +7,14 @@ $(test)_dependencies = TestCommon Ice
 $(test)_client_sources := Client.cpp AllTests.cpp Test.ice TestI.cpp
 
 ifeq ($(os),Darwin)
-$(test)_client_sources += SecureTransportTests.cpp
-$(test)_client_ldflags = -framework Security -framework CoreFoundation
+    $(test)_client_sources += SecureTransportTests.cpp
+    $(test)_client_ldflags = -framework Security -framework CoreFoundation
 endif
 
-#
-# Disable var tracking assignments for Linux with this test
-#
-ifneq ($(linux_id),)
+ifeq ($(os),Linux)
+    $(test)_client_sources += OpenSSLTests.cpp
+    $(test)_client_ldflags = -lssl -lcrypto
```

Comment:
Now that the Ice library links with these libraries (I assume), do we still need to add them to the tests?

Reply:
Yes we link Ice library with OpenSSL. Without this the test linking fails with:

```
/usr/bin/ld: test/IceSSL/configuration/build/aarch64-linux-gnu/shared/pie/OpenSSLTests.o: undefined reference to symbol 'TLS_server_method@@OPENSSL_3.0.0'
/usr/bin/ld: /workspaces/ice/cpp/lib/aarch64-linux-gnu/../../../../../../lib/aarch64-linux-gnu/libssl.so.3: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
```